lighthouse/jobs/reports.py

```diff
@@ -19,39 +20,33 @@
 
 def create_report() -> str:
     """Creates a positve samples on site record using the samples collection and location
-    information from labwhere.
+    information from LabWhere.
 
     Returns:
         str -- filename of report created
     """
     logger.info("Creating positive samples report")
+
     start = time.time()
 
     # get samples collection
-    logger.debug("Getting all positive samples")
     samples_collection = app.data.driver.db.samples
     positive_samples_df = get_all_positive_samples(samples_collection)
 
-    logger.debug("Getting location barcodes from labwhere")
-    labware_to_location_barcode_df = map_labware_to_location(
-        get_distinct_plate_barcodes(samples_collection)
-    )
-
-    logger.debug("Joining location data from labwhere")
-    merged = positive_samples_df.merge(
-        labware_to_location_barcode_df, how="left", on=FIELD_PLATE_BARCODE
-    )
+    logger.info("Getting location barcodes from LabWhere")
+    labware_to_location_barcode_df = map_labware_to_location(get_distinct_plate_barcodes(samples_collection))
 
-    merged = join_samples_declarations(merged)
+    logger.debug("Joining location data from LabWhere")
+    merged = positive_samples_df.merge(labware_to_location_barcode_df, how="left", on=FIELD_PLATE_BARCODE)
 
     merged = add_cherrypicked_column(merged)
 
     report_name, report_path = get_new_report_name_and_path()
 
     logger.info(f"Writing results to {report_path}")
 
-    # Create a Pandas Excel writer using XlsxWriter as the engine
-    writer = pd.ExcelWriter(report_path, engine="xlsxwriter")
+    # Create a Pandas Excel writer using openpyxl as the engine
+    writer = pd.ExcelWriter(report_path, engine="openpyxl")
```

Comment:
Good; so we change to use the recommended writer for pandas, right?

Reply:
Yeah, the other writer was being deprecated I believe.